pkg/koordlet/resmanager/resmanager.go

```diff
@@ -259,3 +271,52 @@ func (r *resmanager) hasSynced() bool {
 
 	return r.nodeSLO != nil && r.nodeSLO.Spec.ResourceUsedThresholdWithBE != nil
 }
+
```

Comment:
better to move evictPod killContainers to util pkg.

Reply:
I'll submit another PR later to refine all functions in util package.

Reply:
remember to add an issue, tks